gravitee-apim-console-webui/src/organization/configuration/tags/org-settings-tags.component.ts

```diff
@@ -15,7 +15,20 @@
  */
 
 import { Component, OnDestroy, OnInit } from '@angular/core';
-import { Subject } from 'rxjs';
+import { combineLatest, Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+import { GroupService } from '../../../services-ngx/group.service';
+import { TagService } from '../../../services-ngx/tag.service';
+import { GioTableWrapperFilters } from '../../../shared/components/gio-table-wrapper/gio-table-wrapper.component';
+import { gioTableFilterCollection } from '../../../shared/components/gio-table-wrapper/gio-table-wrapper.util';
+
+interface TagTableDS {
```

Comment:
`TableDS`feels weird here because there is no real link to any table data source

```suggestion
interface TagVM {
```

Reply:
🤔  I don't know,
 I think it's an object that should be linked to the format expected by the `dataSource` input.
 So it's not just a view model, IMHO 
 This is the Tags object for the Table respecting the input `dataSource` which imposes to have the property corresponding to the defined column